packages/gatsby/src/internal-plugins/internal-data-bridge/gatsby-node.js

```diff
@@ -129,6 +141,24 @@ exports.sourceNodes = ({ createContentDigest, actions, store }) => {
   })
 }
 
+exports.createResolvers = ({ createResolvers }) => {
+  const resolvers = {
+    Site: {
+      buildTime: {
+        type: `Date`,
+        resolve(source, args, context, info) {
```

Comment:
You need to also use original resolver if you want to keep formatting args, see here https://www.gatsbyjs.org/docs/node-apis/#createResolvers

Reply:
Ah, I see what you mean. Do you mean the original resolver from the `SiteBuildTime` node, or a resolver from `Site` node? How would I access these?

Reply:
It's likely that you need to add it `Site` to type definition.

```
type Site {
  buildTime: Date @dateformat
}
```

Reply:
Yeah, just worked that one out!